crypto/sessionkey.go

```diff
@@ -0,0 +1,211 @@
+package crypto
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"io"
+
+	"github.com/ProtonMail/gopenpgp/v2/constants"
+	"github.com/pkg/errors"
+
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/packet"
+)
+
+// SessionKey stores a decrypted session key.
+type SessionKey struct {
+	// The decrypted binary session key.
+	Key []byte
+	// The symmetric encryption algorithm used with this key.
+	Algo string
+}
+
+var symKeyAlgos = map[string]packet.CipherFunction{
+	constants.ThreeDES:  packet.Cipher3DES,
+	constants.TripleDES: packet.Cipher3DES,
```

Comment:
What's the difference between ThreeDES and TripleDES?

Reply:
The underlying crypto is the same, but some refer to it as threeDES ("3des"), other tripleDES ("tripledes")

Reply:
I was interested why we have to keep both and Kubko explained it to me. Thanks!